Library/Styles/CheckoutStyles.swift

```diff
@@ -72,3 +68,8 @@ public let checkoutWhiteBackgroundStyle: ViewStyle = { (view: UIView) in
   view
     |> \.backgroundColor .~ UIColor.white
 }
+
+public let checkoutLayerCardStyleRounded: LayerStyle = { layer in
```

Comment:
For consistency and maybe easier refactoring, could the `Style` be at the end of the constant? (i.e. `checkoutLayerCardRoundedStyle` or `checkoutRoundedCornerStyle`)?